FizzBuzz/UI - single parameter.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'logic'
+
+class UI_single_parameter
+  def initialize
+    @fb_case = Fizz_Buzz.new
+  end
+
+  def fizz_buzz_input
+    puts "Pick a number:"
+    input = gets.chomp
+    input = input.gsub(/,/, '.')
+
+    begin
+      input = Float(input)
+    rescue ArgumentError => e
+      puts "Improper value: #{e.class.name}, try again."
+      @fb_case.print_fizz_buzz(input)
+      exit
+    end
+
+    if input%1 == 0
```

Comment:
better way would be just:

```ruby
if input%1 != 0
  puts "Would you like to round your number #{input}? Y/N"
```

or, more ruby syntax:

```ruby
unless input%1 == 0
```

having empty `if` condition does not contribute much :).